Firmware/LightpackUSB.c

```diff
@@ -211,6 +212,19 @@ void CALLBACK_HID_Device_ProcessHIDReport(USB_ClassInfo_HID_Device_t* const HIDI
     case CMD_NOP:
         break;
 
+
+    // Unofficial commands
+    case CMD_UNOFFICIAL_SET_USBLED:
+
+        g_Settings.isUsbLedEnabled = ReportData_u8[1];
+        if (g_Settings.isUsbLedEnabled) {
+            SET(USBLED);
+        } else {
+            CLR(USBLED);
+        }
+
+        break;
+
```

Comment:
I have never looked into the firmware build. Are the settings persisted across device reboots?